wgsl/index.bs

```diff
@@ -6805,16 +6911,16 @@ textureLoad(t: texture_multisampled_2d<T>, coords: vec2<i32>, sample_index: i32)
 textureLoad(t: texture_multisampled_2d_array<T>, coords: vec2<i32>, array_index: i32, sample_index: i32)-> vec4<T>
 textureLoad(t: texture_depth_2d, coords: vec2<i32>, level: i32) -> f32
 textureLoad(t: texture_depth_2d_array, coords: vec2<i32>, array_index: i32, level: i32) -> f32
-textureLoad(t: [[access(read)]] texture_storage_1d<F>, coords: i32) -> vec4<T>
-textureLoad(t: [[access(read)]] texture_storage_2d<F>, coords: vec2<i32>) -> vec4<T>
-textureLoad(t: [[access(read)]] texture_storage_2d_array<F>, coords: vec2<i32>, array_index: i32) -> vec4<T>
-textureLoad(t: [[access(read)]] texture_storage_3d<F>, coords: vec3<i32>) -> vec4<T>
+textureLoad(t: texture_storage_1d<F,read>, coords: i32) -> vec4<T>
```

Comment:
what's going to happen with read-write textures? are we going to provide overloads for read-write for all of the builtin functions on storage textures?

Reply:
Similar to the proposal for coercing read_write pointers to read pointers, we could do the same thing with textures when read_write textures are added to WGSL. Or, yes, we could just provide overloads.

Reply:
I would say we add overloads for read_write.  We don't need to add coercion for this.